modules/update-service-overview.adoc

```diff
@@ -39,8 +39,13 @@ Only upgrading to a newer version is supported. Reverting or rolling back your c
 
 During the upgrade process, the Machine Config Operator (MCO) applies the new configuration to your cluster machines. The MCO cordons the number of nodes as specified by the `maxUnavailable` field on the machine configuration pool and marks them as unavailable. By default, this value is set to `1`. The MCO then applies the new configuration and reboots the machine.
 
-If you use {op-system-base-full} machines as workers, the MCO does not update the kubelet because you must update the OpenShift API on the machines first. 
+If you use {op-system-base-full} machines as workers, the MCO does not update the kubelet because you must update the OpenShift API on the machines first.
```

Comment:
s/first/before the MCO updates the kubelet.

Reply:
Are you going to implement this comment?

Reply:
@mburke5678 nad @StephenJamesSmith : I will skip this one since I did not update this content and it shows up in the diff because I removed a trailing space. I understand that implementing it would require a QE ack and this is out of scope for the ticket I am addressing in the PR. Hope that is OK.

Reply:
@n1r1 and @JoelSpeed I am concerned with changing _first_ to _before the MCO updates the kubelet_ as Stephen suggested without technical confirmation. As a matter of rule, we avoid using _first_ to avoid confusion that this is the absolutely first thing the user needs to do. 
If you can confirm that this change is correct, we can follow-up in another PR.